opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1073,6 +1094,26 @@ namespace Opm {
         const BlackoilWellModel<TypeTag>&
         wellModel() const { return well_model_; }
 
+        BlackoilAquiferModel<TypeTag>&
+        aquiferModel() { return aquifer_model_; }
+
+        const BlackoilAquiferModel<TypeTag>&
+        aquiferModel() const { return aquifer_model_; }
+
+        int ebosPhaseToFlowCanonicalPhaseIdx( const int phaseIdx ) const
```

Comment:
Is this function used in any way? If not, please remove it.

Reply:
@GitPaean  I have removed this. Thanks